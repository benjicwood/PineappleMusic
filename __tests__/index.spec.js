/* global expect, it */
import 'react-native';
import React from 'react';
import PineappleFront from '../views/container/index.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('exists', () => {
  const tree = renderer.create(
    <PineappleFront />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
