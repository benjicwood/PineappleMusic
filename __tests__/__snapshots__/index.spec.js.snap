exports[`test exists 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
        "overflow": "hidden",
      },
      Object {
        "flex": 1,
      },
    ]
  }>
  <View
    onMoveShouldSetResponder={[Function]}
    onMoveShouldSetResponderCapture={[Function]}
    onResponderEnd={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderReject={[Function]}
    onResponderRelease={[Function]}
    onResponderStart={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    onStartShouldSetResponderCapture={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
        "overflow": "hidden",
      }
    }>
    <View
      onStartShouldSetResponderCapture={[Function]}
      pointerEvents="auto"
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "overflow": "hidden",
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "transform": Array [
              Object {
                "translateX": 0,
              },
              Object {
                "translateY": 0,
              },
              Object {
                "scaleX": 1,
              },
              Object {
                "scaleY": 1,
              },
              Object {
                "rotate": "0deg",
              },
              Object {
                "skewX": "0deg",
              },
              Object {
                "skewY": "0deg",
              },
            ],
          },
          null,
        ]
      }>
      <View
        style={
          Object {
            "flex": 1,
          }
        }>
        <Image
          resizeMode="cover"
          source={1}
          style={
            Array [
              Object {
                "flex": 1,
              },
              Object {
                "height": null,
                "paddingTop": 30,
                "width": null,
              },
            ]
          } />
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          hitSlop={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "borderColor": "blue",
              "borderRadius": 3,
              "borderWidth": 1,
              "flex": 0.3,
              "height": 50,
              "justifyContent": "center",
              "opacity": 1,
            }
          }
          testID={undefined}>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "fontSize": 24,
                "fontWeight": "bold",
              }
            }>
            BAND
          </Text>
        </View>
        <View
          accessibilityComponentType={undefined}
          accessibilityLabel={undefined}
          accessibilityTraits={undefined}
          accessible={true}
          hitSlop={undefined}
          onLayout={undefined}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "borderColor": "blue",
              "borderRadius": 3,
              "borderWidth": 1,
              "flex": 0.3,
              "height": 50,
              "justifyContent": "center",
              "opacity": 1,
            }
          }
          testID={undefined}>
          <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
              Object {
                "fontSize": 24,
                "fontWeight": "bold",
              }
            }>
            MUSICIAN
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
